tests/test_interpreters.py

```diff
@@ -6,24 +6,20 @@
 import py
 import pytest
 
+import tox
 from tox.config import get_plugin_manager
-from tox.interpreters import ExecFailed
-from tox.interpreters import InterpreterInfo
-from tox.interpreters import Interpreters
-from tox.interpreters import NoInterpreterInfo
-from tox.interpreters import pyinfo
-from tox.interpreters import run_and_get_interpreter_info
-from tox.interpreters import sitepackagesdir
-from tox.interpreters import tox_get_python_executable
-
-
-@pytest.fixture
-def interpreters():
+from tox.interpreters import (
```

Comment:
I'm not to keen on the paranthesis? isn't that redundant? also why indent it and not just start it on this line, that way would fit in two lines e.g. here

Reply:
ping again on this :-)